app/src/main/res/values/colors.xml

```diff
@@ -16,4 +16,6 @@
   <color name="dialog_dim">#4D000000</color>
   <color name="track_filters_close_button_tint">#4D000000</color>
 
+  <color name="schedule_empty_view_text_color">#4d606060</color>
```

Comment:
You sure it's not a colour we've used elsewhere? It would seem weird to me it's not on our palette

Reply:
I'll look for it, the other empty state uses a darker gray

Reply:
I think we should use that same one — the colour in the designs is likely different because those designs have a blue colour scheme :)

Reply:
I think the favorites one is the one to be changed, the new designs are indeed different